zerver/views/webhooks/freshdesk.py

```diff
@@ -41,10 +50,13 @@ def property_name(property, index):
     else:
         raise ValueError("Unknown property")
 
+
 def parse_freshdesk_event(event_string):
-    # These are always of the form "{ticket_action:created}" or
-    # "{status:{from:4,to:6}}". Note the lack of string quoting: this isn't
-    # valid JSON so we have to parse it ourselves.
+    # type: (str) -> Union[List[str], Tuple[str, str, str]]
```

Comment:
Do we really need to support both List[str] and Tuple[str, str, str] here?  Seems like it could be the case we should just switch this stuff to use one or the other...